ui/admin/app/templates/scopes/scope/auth-methods/index.hbs

```diff
@@ -119,35 +119,15 @@
           {{#each @model as |authMethod|}}
             <body.row as |row|>
               <row.headerCell>
-                {{!
-                  Show link to auth method route if non-OIDC and when
-                  oidc-crud feature is enabled, otherwise show just the name
-                  (when the method is OIDC and oidc-crud is disabled)
-                }}
-                {{#if authMethod.isOIDC}}
```

Comment:
Isn't this check still needed?

Reply:
Yes it is! Good catch.

Reply:
Actually, I am not sure now. I checked out the branch compared to main and they both function the same way. I believe we do want to show OIDC auth methods. I am fairly certain that was been in production for a while. @gsusmi or @randallmorey do you have more input for this.

Reply:
Looking through this again, other than checking for capability to read an auth-method, auth-method type checks shouldn't be needed. Sorry for causing the confusion.

Reply:
No worries!!